extension.toml

```diff
@@ -1,6 +1,6 @@
 id = "catppuccin-icons"
 name = "Catppuccin Icons"
-version = "1.18.0"
+version = "1.20.0"
```

Comment:
Copy/paste error?

Reply:
ahh yeah. search n replace error to sync the two tags